### PR TITLE
Move building info to employee module

### PR DESCRIPTION
**Pull request by var-kyle**
Lets you import the building info in the employee module rather than having to install the Contacts module. The "Location" option is located in the configuration drop down.

resolves gcdevops/HRWhiteListing#199

### CODE REVIEW

**Comment by var-kyle** [comment]
As discussed, the gitignore can remain the way it it.

I have removed the **view_location_form** because it was not actually used.

The bottom section has been removed as well.

I have also updated the search to be more relevant for this view.

File: add-ons/hr_modifier/views/hr_location_views.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <record id="view_location_filter" model="ir.ui.view">
+            <field name="name">hr.location.search</field>
+            <field name="model">res.partner</field>
+            <field name="arch" type="xml">
+                <search string="Location">
+                    <field name="name" string="Name"/>
+                    <field name="city" string="City"/>
+                    <field name="country_id" string="Country"/>
+                    <field name="state_id" string="Province"/>
+                    <field name="stree" string="Street"/>
```
Comment (will0684) [inline add-ons/hr_modifier/views/hr_location_views.xml:36]: Field name is broken here, should be "street". Breaks install of module.

Comment (will0684) [reply add-ons/hr_modifier/views/hr_location_views.xml:36]: Besides this it works well!

Comment (var-kyle) [reply add-ons/hr_modifier/views/hr_location_views.xml]: Typo is fixed.

**Comment by gabrielcossette** [comment]
- Changed behaviour so that companies are created by default (Create button) and adjusted the modification of the Contact form.
- Trying to show only companies in the Locations list with a record rule, but it is currently not working only for Senior Management group. Will test more.

If you want to see my edits, only look at my last 2 commits above. The 2 others were reverted.

**Comment by gabrielcossette** [comment]
Solution to show only companies in Locations list was not record rule but rather a domain on the Action Window.

@var-kyle Feel free to review but PR should be good to go. Great job!

**Pull request merged**